submit.py

```diff
@@ -43,6 +43,11 @@ def input(str):
 Metadata = namedtuple("Metadata", ['assignment_key', 'name', 'part_data'])
 Part = namedtuple("Part", ['id', 'input_file', 'solver_file', 'name'])
 
+email_prompt = 'User Name (e-mail address): '
+token_prompt = 'Submission Token (from the assignment page): '
+email_env_var = 'EMAIL_ADDRESS'
+token_env_var = 'DOTOKEN'
```

Comment:
I agree DO_TOKEN will be clearer. I'll change it to DO_TOKEN when I update the pr.